### PR TITLE
adding example models

### PR DESCRIPTION
**Pull request by kathleen547**
Two models added

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class CNN(nn.Module):
+
+    # define init method
+    # define layers
+    def __init__(self, hiddenLayer=100):
+
```
Comment (woocash2) [inline model.py:6]: Remove empty line

File: model.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        super().__init__()
+        self.conv1 = nn.Conv2d(3, 6, 5)
+        self.pool = nn.MaxPool2d(2, 2)
+        self.fc1 = nn.Linear(6 * 14 * 14, hiddenLayer)
+        self.out = nn.Linear(hiddenLayer, 10)
+        self.act = nn.ReLU()
+
+    def forward(self, x):
+
```
Comment (woocash2) [inline model.py:15]: rm empty line

File: model.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+class CNN(nn.Module):
```
Comment (woocash2) [inline model.py:1]: Have you tested that both of these networks work (i.e. they learn, and have reasonable accuracy on the test set)?

Comment (kathleen547) [reply model.py:1]: First model has accuracy 37%, second 36%
